packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 51.16.0 - June 3, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add Java support for Functions debugging ([PR #3273](https://github.com/forcedotcom/salesforcedx-vscode/pull/3273))
```

Comment:
```suggestion
- Added Java support for Functions debugging ([PR #3273](https://github.com/forcedotcom/salesforcedx-vscode/pull/3273))
```

Reply:
We went over a similar comment couple of weeks ago and decided to use present tense.
I've committed the other changes though.